content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not work affect runtime of Prisma CLI or Prisma Client at all).
 
-For example:
-If you are on macOS, the binary target is `darwin` & AWS Lambda target is `rhel-openssl-1.0.x`
+This is to be used if you deploy to a specific platform via an upload of a local project including dependencies and your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`) and makes sure the target engine files are also downloaded.
 
 ```terminal
 PRISMA_CLI_BINARY_TARGETS=darwin,rhel-openssl-1.0.x npm install
 ```
 
-This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets) which enables you to define the same setting for Prisma Client
+This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets) which enables you to define the same setting for Prisma Client.
```

Comment:
"which" vs "that" issue... we can add a comma and say "... equivalent for the `binaryTargets` property of the `generator` block, which..." or we can use "that" ("... equivalent for the `binaryTargets` property of the `generator` block that enables you..."). In this sentence I prefer "which" with a comma. :-)

Reply:
```suggestion
This is the Prisma CLI equivalent for the [`binaryTargets` property of the `generator` block](/concepts/components/prisma-schema/generators#binary-targets), which enables you to define the same setting for Prisma Client.
```